core/src/main/resources/openapi-specs/eth2/signing/schemas.yaml

```diff
@@ -393,6 +393,7 @@ components:
       properties:
         beacon_block:
           anyOf:
+            - $ref: '#/components/schemas/BlockRequestBellatrix'
```

Comment:
Is this order significant? If not would be kind of nice to have these in order of the forks. Very minor quibble though.

Reply:
no, its not. Updated.